### ISSUE TITLE
parse server update doesn't works

### ISSUE THREAD

**Issue by azielazny** [bitnami/bitnami-docker-parse#49]
**Description**

 I have a problem with updating the parse server from the docker-compose level. When I change `image: 'bitnami/parse:latest'` to `image: 'bitnami/parse:3.1.3'` or `image: 'bitnami/parse:3'` then all the time Parse Dashboard on the browser shows me version 2.8.4. How to update parse-server version to the latest version?

On the 2.8.4 version of the parser I have a problem with the stability of the container. When I try to add a user(username+password) from Parse Dashboard on the browser, then I get an error.
```parse_1            | error: Error generating response. ParseError { code: 201, message: 'password is required' } code=201, message=password is required
parse_1            | error: password is required code=201, message=password is required
parse_1            | /opt/bitnami/parse/node_modules/parse-server/lib/ParseServer.js:218
parse_1            |           throw err;
parse_1            |           ^
...
exited with code 7
```

From the level of the Android Studio application I can add a user, but when I try to add an existing user a second time, then he throws me the same error with `exited with code 7` in docker.

This may be an older version bug, but I can't update parse server.

**Describe the results you expected:**

For example, I can get information about trying to add a user again, but why an exception is thrown and a container being killed.
I want to be able to add a user from Parse Dashboard in the browser without throws and exit from docker.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

- Output of `docker info`:

```
λ docker info
Containers: 3
 Running: 0
 Paused: 0
 Stopped: 3
Images: 3
Server Version: 18.09.0
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 468a545b9edcd5932818eb9de8e72413e616e86e
runc version: 69663f0bd4b60df09991c08812a60108003fa340
init version: fec3683
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.9.125-linuxkit
Operating System: Docker for Windows
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 1.934GiB
Name: linuxkit-00155da09201
ID: L3UI:DJCX:JNYJ:AMMH:HDXD:MXMP:HEBZ:U3CE:MKKR:XTF2:6CY5:ODBD
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 22
 Goroutines: 47
 System Time: 2019-01-05T16:03:57.1600559Z
 EventsListeners: 1
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
Product License: Community Engine
```

- Output of `docker-compose version` (if applicable):

```
1.23.2
```

**My docker-compose.yml:**

```version: '2'
services:
  mongodb:
    image: 'bitnami/mongodb:latest'
    volumes:
      - 'mongodb_data:/bitnami'
  parse:
    image: 'bitnami/parse:3.1.3'
    environment:
      - PARSE_HOST=83.xxx.xxx.82
    ports:
      - '1338:1337'
    volumes:
      - 'parse_data:/bitnami'
    depends_on:
      - mongodb
  parse-dashboard:
    image: 'bitnami/parse-dashboard:latest'
    environment:
      - PARSE_HOST=83.xxx.xxx.82
      - PARSE_PORT_NUMBER=1338
    labels:
      kompose.service.type: nodeport
    ports:
      - '80:4040'
    volumes:
      - 'parse_dashboard_data:/bitnami'
    depends_on:
      - parse
volumes:
  mongodb_data:
    driver: local
  parse_data:
    driver: local
  parse_dashboard_data:
    driver: local
```

**Comment by carrodher** [comment]
Sorry for the delay in our reply, we created an internal task to investigate both issues, we will post here the advances found. Thank you!

**Comment by alemorcuq** [comment]
Hi, @azielazny.

Sorry for the delay. The issue was fixed in the `3.2.3-r2` release. It should work now. Let us now if you are still encountering any problems.
